mysql/infoschema.go

```diff
@@ -22,6 +22,7 @@ import (
 
 	"github.com/cloudspannerecosystem/harbourbridge/internal"
 	"github.com/cloudspannerecosystem/harbourbridge/schema"
+	spangres "github.com/cloudspannerecosystem/harbourbridge/spangres"
```

Comment:
nit: correct me if I am wrong but I don't think the spangres alias is adding something here since the module name is spangres already.

Reply:
Similar suggestion in pg, pgdump, and mysqldump

Reply:
Done